api/routes.js

```diff
@@ -3,13 +3,15 @@ const controllers = require('./controllers');
 const auth = require('./middleware').auth;
 const helpers = require('./helpers');
 const multer = require('multer');
+const memStorage = multer.memoryStorage();
 
 const upload = multer({
-  storage: multer.memoryStorage(),
-  fileFilter: helpers.filefilter,
+  storage: memStorage,
```

Comment:
Reviewed and market security warning as safe

Reply:
Thanks!